### PR TITLE
Feature/owner service

### PR DESCRIPTION
**Pull request by Myheartdies**
Implementation for OwnerService, with modification to IOwnerService

### CODE REVIEW

File: AggieRent/Services/OwnerService.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    private readonly IOwnerRepository _ownerRepository = ownerRepository;
+
+    public Owner? GetOwnerById(string id)
+    {
+        return _ownerRepository.Get(id);
+    }
+
+    public IEnumerable<Owner> GetOwners()
+    {
+        return _ownerRepository.GetAll();
```
Comment (choutianxius) [inline AggieRent/Services/OwnerService.cs:16]: Use .ToList here to load in memory

File: AggieRent/Services/OwnerService.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+using AggieRent.Models;
+using AggieRent.Services;
+
+public class OwnerService(IOwnerRepository ownerRepository) : IOwnerService
+{
+    private readonly IOwnerRepository _ownerRepository = ownerRepository;
+
+    public Owner? GetOwnerById(string id)
+    {
+        return _ownerRepository.Get(id);
```
Comment (choutianxius) [inline AggieRent/Services/OwnerService.cs:11]: GetVerbose

**Review by choutianxius — CHANGES_REQUESTED** [review]

**Comment by choutianxius** [comment]
What do you mean by 'partial'?

**Comment by Myheartdies** [comment]
> What do you mean by 'partial'?

I had a gut feeling that these should not be all there are to OwnerService but couldn't think of what else should be added so I said "partial". Upon rethinking I realize that this already has all the CRUD actions currently needed. 
I will remove the 'partial' description from the PR

**Review by choutianxius — APPROVED** [review]

**Pull request merged**